.github/workflows/tests_linux.yml

```diff
@@ -590,7 +591,7 @@ jobs:
         run: |
           cd main/
           DEVICENAME=`echo ${{ matrix.pl_backend }} | sed "s/_/./g"`
-          PL_DEVICE=${DEVICENAME} python -m pytest tests/ $COVERAGE_FLAGS
+          PL_DEVICE=${DEVICENAME} python -m pytest tests/ -k "not test_native_mcm" $COVERAGE_FLAGS
```

Comment:
Do we need to do that, since we have a [device name control ](https://github.com/PennyLaneAI/pennylane-lightning/pull/621/files#diff-cd218047f28fcee04400b9e924bdc451f050d553f4867212944603325d909d3fR28-R29)already in place?

Reply:
This is not required, just to be safe.